account-kit/infra/src/middleware/gasManager.ts

```diff
@@ -408,7 +408,7 @@ const generateSignedPermit = async <
 
   let maxAmountToken = maxUint256;
 
-  if (policyToken.maxTokenAmount) {
+  if (policyToken.maxTokenAmount !== undefined) {
```

Comment:
❔ What about the case that maxTokenAmount == null

Reply:
new bee to TS: this is typed as 'undefined | bigInt', is that possible to assign 'null' to it?

Reply:
i think best practice is to check for `policyToken.maxTokenAmount != null`, which checks for `null` or `undefined`.

i believe the main reason to make the change here is jjust in case someone makes a change later where it's nullable and doesn't notice that you are strictly checking for `undefined` here.